code/modules/mob/living/simple_animal/abnormality/_tools/zayin/wishwell.dm

```diff
@@ -0,0 +1,250 @@
+		/obj/item/clothing/under/suit/lobotomy/control,
+		/obj/item/clothing/under/suit/lobotomy/information,
+		/obj/item/clothing/under/suit/lobotomy/safety,
+		/obj/item/clothing/under/suit/lobotomy/training,
+		/obj/item/clothing/under/suit/lobotomy/command,
+		/obj/item/clothing/under/suit/lobotomy/discipline,
+		/obj/item/clothing/under/suit/lobotomy/welfare,
+		/obj/item/clothing/under/suit/lobotomy/extraction,
+		/obj/item/clothing/under/suit/lobotomy/records
+		)
+	var/list/baditem = list(
+		/obj/item/poster/random_contraband,
+		/obj/item/poster/random_official,
+		/obj/item/camera,
+		/obj/item/light/bulb,
+		/obj/item/light/tube,
+		/obj/item/toy/eightball,
+		/obj/item/rack_parts,
+		/obj/item/clothing/under/color/rainbow,
+		/obj/item/coin/silver
+		)
+	var/list/trash = list(
+		/obj/item/toy/plush/fumo, //Needs no explanation
+		/obj/item/trash/raisins,
+		/obj/item/trash/candy,
+		/obj/item/trash/cheesie,
+		/obj/item/trash/chips,
+		/obj/item/trash/popcorn,
+		/obj/item/trash/sosjerky,
+		/obj/item/trash/plate,
+		/obj/item/trash/pistachios,
+		/obj/item/paper/crumpled,
+		/obj/effect/decal/cleanable/ash,
+		/obj/item/cigbutt,
+		/obj/item/food/urinalcake,
+		/obj/item/shard,
+		/obj/item/reagent_containers/pill/maintenance
+		)
+
+//Potential threats
+	var/list/dawn = list(
+		/mob/living/simple_animal/hostile/ordeal/green_bot,
+		/mob/living/simple_animal/hostile/ordeal/indigo_dawn,
+		/mob/living/simple_animal/hostile/ordeal/violet_fruit
+		)
+	var/list/noon = list(
+		/mob/living/simple_animal/hostile/ordeal/green_bot_big,
+		/mob/living/simple_animal/hostile/ordeal/indigo_noon,
+		/mob/living/simple_animal/hostile/slime
+		)
+	var/list/dusk = list(
+		/mob/living/simple_animal/hostile/ordeal/amber_dusk,
+		/mob/living/simple_animal/hostile/ordeal/green_dusk,
+		/mob/living/simple_animal/hostile/mini_censored,
+		/mob/living/simple_animal/hostile/slime/big
+		)
+	var/list/midnight = list(//TODO: Wait until reasonable threats exist. These guys are too easy and midnights are too hard. Egor said no fixers.
+		/mob/living/simple_animal/hostile/ordeal/indigo_dusk/red,
+		/mob/living/simple_animal/hostile/ordeal/indigo_dusk/white,
+		/mob/living/simple_animal/hostile/ordeal/indigo_dusk/black,
+		/mob/living/simple_animal/hostile/ordeal/indigo_dusk/pale
+		)
+
+//This proc removes the need to copypaste every single armor and weapon into a list.
+/obj/structure/toolabnormality/wishwell/Initialize()
+	..()
+
+//Sorts them into their lists
+	for(var/path in subtypesof(/datum/ego_datum))
+		var/datum/ego_datum/ego = new path()
+		switch(ego.cost)
+			if (250)
+				superEGO+=ego.item_path
+			if (100)
+				alephitem+=ego.item_path
+			if (50)
+				wawitem+=ego.item_path
+			if (35 to 40)
+				heitem+=ego.item_path
+			if (20 to 25)
+				tethitem+=ego.item_path
+			if (12)
+				zayinitem+=ego.item_path
+
+//End of loot lists
+/obj/structure/toolabnormality/wishwell/attackby(obj/item/I, mob/living/carbon/human/user)
+	..()
```

Comment:
Remove this line

Reply:
Removing this line causes far more issues than it solves, but I can try to find an alternate solution.

Reply:
After testing the change doesn't seem feasible with respect to the rest of the code at the time of writing.